docs/upgrade-backup.md

```diff
@@ -0,0 +1,40 @@
+# Restore 8.0 backup to 8.4 cluster
+
+Use Percona XtraBackup to back up the source server data and restore the data to a target server, and then upgrade the server to a different version of Percona XtraDB Cluster.
+
+## Restore a database with a different server version
+
+Review [Upgrade Percona XtraDB cluster](upgrade-guide.md).
+
+Upgrade the nodes one at a time. The primary node should be the last node to be upgraded. The following steps are required on each node.
+
+1. Install the same database version as the source server (the backup version) on the target server.
```

Comment:
I don't think steps 1 and 2 are necessary. We can start from point 3.
So: restore the backup, start the server with desired 8.4 version, enjoy.
@Tusamarco , any comments?